README.md

```diff
@@ -26,49 +10,112 @@ This is a Java Spring Boot Web API reference application designed to "fork and c
+                                        -Dmaven.legacyLocalRepo=true
+ -N,--non-recursive                     Do not recurse into sub-projects
+ -npr,--no-plugin-registry              Ineffective, only kept for
+                                        backward compatibility
+ -npu,--no-plugin-updates               Ineffective, only kept for
+                                        backward compatibility
+ -nsu,--no-snapshot-updates             Suppress SNAPSHOT updates
+ -ntp,--no-transfer-progress            Do not display transfer progress
+                                        when downloading or uploading
+ -o,--offline                           Work offline
+ -P,--activate-profiles <arg>           Comma-delimited list of profiles
+                                        to activate
+ -pl,--projects <arg>                   Comma-delimited list of specified
+                                        reactor projects to build instead
+                                        of all projects. A project can be
+                                        specified by [groupId]:artifactId
+                                        or by its relative path
+ -q,--quiet                             Quiet output - only show errors
+ -rf,--resume-from <arg>                Resume reactor from specified
+                                        project
+ -s,--settings <arg>                    Alternate path for the user
+                                        settings file
+ -t,--toolchains <arg>                  Alternate path for the user
+                                        toolchains file
+ -T,--threads <arg>                     Thread count, for instance 2.0C
+                                        where C is core multiplied
+ -U,--update-snapshots                  Forces a check for missing
+                                        releases and updated snapshots on
+                                        remote repositories
+ -up,--update-plugins                   Ineffective, only kept for
+                                        backward compatibility
+ -v,--version                           Display version information
+ -V,--show-version                      Display version information
+                                        WITHOUT stopping build
+ -X,--debug                             Produce execution debug output
+
+ ```
+
+## Run the Application
 
-```bash
+### Using Visual Studio Codespaces
 
-# make sure you are logged into Azure
-az account show
+> Visual Studio Codespaces is the easiest way to evaluate ngsa-java.
 
-# if not, log in
-az login
+1. Set up Codespaces from the GitHub repo
 
-```
+2. Input credentials in CosmosUrl and CosmosKey files within secrets folder (Create files if necessary)
 
-### Verify Key Vault Access
+3. Run the application
 
 ```bash
 
-# verify you have access to Key Vault
-az keyvault secret show --name CosmosDatabase --vault-name $He_Name
+# run the application
+mvn spring-boot:run
```

Comment:
Should use    `mvn clean spring-boot:run` instead  to be consistent with what is logged in usage